pwa/src/templates/templateParts/actions/ActionDetailsTemplate.tsx

```diff
@@ -1,13 +1,88 @@
 import * as React from "react";
-import { Heading1 } from "@gemeente-denhaag/components-react";
+import * as styles from "./ActionDetailsTemplate.module.css";
+import { Heading1, Heading2 } from "@gemeente-denhaag/components-react";
 import { useTranslation } from "react-i18next";
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from "@gemeente-denhaag/table";
+import { useAction } from "../../../hooks/action";
+import { QueryClient } from "react-query";
+import { Container } from "@conduction/components";
 
-export const ActionsDetailTemplate: React.FC = () => {
+interface ActionDetailsTemplateProps {
+  actionId: string;
+}
+
+export const ActionsDetailTemplate: React.FC<ActionDetailsTemplateProps> = ({ actionId }) => {
   const { t } = useTranslation();
 
+  const queryClient = new QueryClient();
+  const _useActions = useAction(queryClient);
+  const getActions = _useActions.getOne(actionId);
+
   return (
-    <div>
+    <Container layoutClassName={styles.container}>
```

Comment:
I think it would be nice to have a back button here like so:
~~~~
      <div onClick={() => navigate("/actions")}>
        <Link icon={<ArrowLeftIcon />} iconAlign="start">
          {t("Back to actions")}
        </Link>
      </div>
~~~~